sprint-organizer.js

```diff
@@ -0,0 +1,394 @@
+/*
+    This is a recipe to organize MTC Sprints
+
+    - discover existing sprint
+    - take an end-of-sprint snapshot
+    - create new sprint
+    - move all in-progress / to-do issues from the previous sprint to the new sprint
+    - take a new-born sprint snapshot
+    - wait for folks to add new issues
+    - take a beginning-of-sprint snapshot
+    - repeat
+*/
+const githubClient = require('./gh-client')
+const googleDriveClient = require('./drive-client')
+const fs = require('fs')
+var xlsx = require('xlsx');
+const emailer = require('./emailer')
```

Comment:
+1 to breaking off the clients and emailer. I was hoping we could build this so it could be used by other teams, this will help.